src/routes/capacity.js

```diff
@@ -81,7 +85,7 @@ router.get(
       const capacities = getAllProductCapacities(store, period);
 
       const response = capacities.map(capacity => formatCapacityResult(capacity));
-      console.log(JSON.stringify(capacities, null, 2));
+      console.debug('Response: ', inspect(response, { depth: null }));
```

Comment:
made the `/capacity` response to be also debug as it could be too verbose logging it everytime as it prints capacities for ALL products